code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -4263,7 +4263,7 @@
 	description = "A chicken before it could become a chicken."
 	nutriment_factor = 15 * REAGENTS_METABOLISM
 	reagent_state = REAGENT_STATE_LIQUID
-	color = "#FFFACD" //LEMONCHIFFON
+	color = "#ffcd42"
```

Comment:
This should be #03fc30

Reply:
remove your green eggs and ham propaganda from the site of my holy mission